src/controllers/update/check.your.answers.controller.ts

```diff
@@ -0,0 +1,57 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import { getApplicationData } from "../../utils/application.data";
+import { ApplicationData } from "../../model";
+import { Session } from "@companieshouse/node-session-handler";
+
+import { createOverseasEntity } from "../../service/overseas.entities.service";
+import { closeTransaction, postTransaction } from "../../service/transaction.service";
+import { startPaymentsSession } from "../../service/payment.service";
+
+import {
+  CHS_URL,
+  OVERSEAS_ENTITY_QUERY_URL,
+  UPDATE_AN_OVERSEAS_ENTITY_URL,
+  UPDATE_CHECK_YOUR_ANSWERS_PAGE
+} from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${UPDATE_CHECK_YOUR_ANSWERS_PAGE}`);
+
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    return res.render(UPDATE_CHECK_YOUR_ANSWERS_PAGE, {
+      backLinkUrl: OVERSEAS_ENTITY_QUERY_URL, // TO DO: update whenever UAR-106 merged
+      templateName: UPDATE_CHECK_YOUR_ANSWERS_PAGE,
+      appData
```

Comment:
why need pass `appData` 
i did not found any usage in template

Reply:
On current moment in `views/update/update-check-your-answers.html` you do not use any `addData.<field>`
so there is no reason to pass do not used object to template.
better do not keep do not used source code

Reply:
done